desktop/src/renderer/src/layouts/PageLayout.tsx

```diff
@@ -0,0 +1,16 @@
+import { Grid2 as Grid } from "@mui/material";
+
+export const PageLayout = ({ children }: { children: React.ReactNode }) => {
+	return (
+		<Grid container>
+			<Grid size={2} />
+			<Grid
+				size={8}
+				sx={{ paddingTop: 3 }}
+			>
+				{children}
+			</Grid>
+      <Grid size={2} />
```

Comment:
This layout is supposed to be different for file list. On the file list page its supposed to have a wider layout to fit more of the table columns. Originally it was Grid sizes 0.5, 11, and 0.5 instead of the normal 2, 8, 2